tests/HttpExceptionTests.php

```diff
@@ -0,0 +1,30 @@
+<?php
+use PHPUnit\Framework\TestCase;
+use Tests\Tnapf\Router\MissingDescriptionHttpException;
+use Tests\Tnapf\Router\MissingPhraseHttpException;
+use Tnapf\Router\Exceptions\HttpInternalServerError;
+
+class HttpExceptionTests extends TestCase {
+    public function testMissingPhrase(): void
+    {
+        $this->expectException(\RuntimeException::class);
+
+        MissingPhraseHttpException::buildJsonResponse();
```

Comment:
What are you testing here? This seems like it is only there for test coverage

Reply:
Thats not how you should be testing this. now you have 2 classes in your main codebase that will always error out.

Reply:
They're in the tests directory though?

Reply:
anonymous class

Reply:
Please elaborate?

Reply:
```php
$missingPhraseException = new class extends Whatever {
};

$missingPhraseException::buildJsonResponse();
```

Reply:
> anonymous class

My fault I kept reading anonymous function. Yeah an anonymous class would be great for this